src/queueserver.ts

```diff
@@ -1,7 +1,7 @@
 import axios from "axios";
 
 var axiosInstance = axios.create({
-    baseURL: '/qs',
+    baseURL: process.env.REACT_APP_API_PREFIX,
```

Comment:
Where does the default value of this get set?

Reply:
In .env in the repo

Reply:
Ah, already there hence why it is not in the diff :+1:

Reply:
Yes, I had been meaning to use it and got distracted by something else.